edk2toolext/tests/test_nuget_dependency.py

```diff
@@ -125,10 +125,18 @@ def test_normalize_version(self):
         version3 = "6"
         proper_version3 = "6.0.0"
         self.assertEqual(proper_version3, NugetDependency.normalize_version(version3))
+        version4 = "6-beta"
+        proper_version4 = "6.0.0-beta"
+        self.assertEqual(proper_version4, NugetDependency.normalize_version(version4))
         # try some bad cases
-        version4 = "not a number"
         with self.assertRaises(ValueError):
-            NugetDependency.normalize_version(version4)
+            NugetDependency.normalize_version("not a number")
+        with self.assertRaises(ValueError):
+            NugetDependency.normalize_version("6.0-beta-beta")
+        with self.assertRaises(ValueError):
+            NugetDependency.normalize_version("6.0-bad")
```

Comment:
Can we add a test case for "3.2.1.-beta"?
Dash immediately after .
How about ---?

Reply:
Done